### PR TITLE
[tools] Add test suite list files.

### PR DESCRIPTION
**Pull request by qiuzhong**
Add a test suite list file named PLATFORMS for master for better maintaince.
Move the test suite list file for WTS to tools/build/released_suites.
Sort the test suite names in PLATFORMS by alphabetical order.

http://crosswalk-project.org/jira/browse/XWALK-4617

### CODE REVIEW

**Comment by qiuzhong** [comment]
Please review this PR.

File: tools/build/suites/weekly/canary_list [hunk 1]
```diff
@@ -0,0 +1,203 @@
+AIOLIST="webapi-noneservice-tests
```
Comment (zqzhang) [inline tools/build/suites/weekly/canary_list]: Do you have list like `beta/stable_list` in the master branch?

If not, why don't use the naming as https://github.com/crosswalk-project/crosswalk-test-suite/pull/2497#issuecomment-121534484

> `tools/build/suites/WEEKLY`, `tools/build/suites/WTS`

**Comment by jennycao** [comment]
@qiuzhong @zqzhang since WTS release is also a part of weekly release.
So I suggest the folder and release list name is better to be
build/suites/platforms_release_list 
build/suites/wts_release_list

**Comment by zqzhang** [comment]
> So I suggest the folder and release list name is better to be
> build/suites/platforms_release_list
> build/suites/wts_release_list

The `_release_list` here is not necessary.

Another because these 2 files are used as constant variable by shell scripts, better use upper letter:

- build/suites/PLATFORM
- build/suites/WTS

**Comment by jennycao** [comment]
Agree：）
    build/released_suites/PLATFORMS (since this list cover several platforms)
    build/released_suites/WTS

**Comment by zqzhang** [comment]
> build/released_suites/PLATFORMS (since this list cover several platforms)
> build/released_suites/WTS

Again, `released_` is not necessary because it is a sub-directory of `build/`.

**Comment by jennycao** [comment]
No, Build != Release
The build folder save the script for building all packages, but the released suites lists are not include all test suites, 
Another difference for Build and Release:
We have nightly build each day but only have release once 1 week.

**Comment by qiuzhong** [comment]
@zqzhang Before you suggested to use tools/build/suites/WEEKLY, now tools/build/suites/PLATFORMS?

**Comment by zqzhang** [comment]
LGTM, but still missing the script that reads `tools/build/released_suites/PLATFORMS`.

**Comment by qiuzhong** [comment]
The script that reads tools/build/released_suites/PLATFORMS is for nightly build only. This script is highly depended on the environment of nightly build server. I don't think it's part of crosswalk-test-suite. So I don't think it's a good idea to publish on github.

**Comment by jennycao** [comment]
LGTM

**Comment by jennycao** [comment]
LGTM

**Pull request merged**